### PR TITLE
use for_js, don't export OCaml id

### PR DESCRIPTION
**Pull request by jchavarri**

### CODE REVIEW

File: ppx/ast_attributes.ml [hunk 1]
```diff
@@ -423,7 +423,10 @@ let is_mel_as : attr -> bool =
   warn_if_bs ~loc txt;
   txt = "mel.as" || txt = "bs.as" || txt = "as"
 
-let has_mel_as_payload (attrs : t) = Ext_list.find_first attrs is_mel_as
+let has_mel_for_js_payload (attrs : t) =
+  Ext_list.find_first attrs (fun { attr_name = { txt; loc }; _ } ->
+      warn_if_bs ~loc txt;
+      txt = "mel.for_js")
```
Comment (davesnx) [inline ppx/ast_attributes.ml:429]: I think a mel.export might make more sense here?

Comment (anmonteiro) [reply ppx/ast_attributes.ml:429]: we chatted offline about reusing `mel.as`.

Comment (davesnx) [reply ppx/ast_attributes.ml:429]: Make sense also

**Comment by anmonteiro** [comment]
Thanks, will continue in #714

**Pull request merged**